packages/@o3r/components/testing/setup-jest.ts

```diff
@@ -1,2 +1,2 @@
+import '@o3r/test-helpers/src/setup-jest';
```

Comment:
please avoid deep import, you can define a mapping in the `@o3r/test-helpers` package.json's `exports` properties